state_machine/flight_manager.py

```diff
@@ -37,9 +38,14 @@ class FlightManager:
     def __init__(self) -> None:
         """Initialize a flight manager object."""
 
-    def start_manager(self) -> None:
-        """Test running the state machine in a separate process."""
+    def start_manager(self, simflag: bool) -> None:
+        """Test running the state machine in a separate process.
+        Sets the drone address to the simulation or physical address."""
```

Comment:
Docstring should be reformatted to adhere to Numpy standard. You need to include a "Parameters" section for your new variable sim_flag